webf/lib/src/bridge/to_native.dart

```diff
@@ -127,43 +127,67 @@ final DartParseHTML _parseHTML =
 
 int _anonymousScriptEvaluationId = 0;
 
-void evaluateScripts(int contextId, String code, {String? url, int line = 0}) {
+class ScriptByteCode {
+  ScriptByteCode();
+  late Uint8List bytes;
+}
+
+Future<bool> evaluateScripts(int contextId, String code, {String? url, int line = 0}) async {
   if (WebFController.getControllerOfJSContextId(contextId) == null) {
-    return;
+    return false;
   }
   // Assign `vm://$id` for no url (anonymous scripts).
   if (url == null) {
     url = 'vm://$_anonymousScriptEvaluationId';
     _anonymousScriptEvaluationId++;
   }
 
-  Pointer<NativeString> nativeString = stringToNativeString(code);
-  Pointer<Utf8> _url = url.toNativeUtf8();
-  try {
-    assert(_allocatedPages.containsKey(contextId));
-    _evaluateScripts(_allocatedPages[contextId]!, nativeString, _url, line);
-  } catch (e, stack) {
-    print('$e\n$stack');
+  QuickJSByteCodeCacheObject cacheObject = await QuickJSByteCodeCache.getCacheObject(code);
```

Comment:
I think we should add an option to disable the cache or a way to clear caches. If not, The memory leak would occur.